From a2553c203d8b8257dea1d2e2139b220935587144 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Sat, 25 Oct 2014 20:03:25 +0200
Subject: [PATCH] Add AC_CONFIG_MACRO_DIR to configure.in

Without AC_CONFIG_MACRO_DIR, when autoreconfiguring the package, it
cannot find AM_PATH_EXPAT which is defined in config/expat.m4.

Submitted upstream: https://github.com/rcmadruga/log4c-localtime/pull/1

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 configure.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.in b/configure.in
index dbf11e9..769b204 100644
--- a/configure.in
+++ b/configure.in
@@ -5,6 +5,7 @@ AC_PREREQ(2.57)
 AC_INIT
 AC_CONFIG_SRCDIR([configure.in])
 AC_CONFIG_AUX_DIR(config)
+AC_CONFIG_MACRO_DIR([config])
 AM_CONFIG_HEADER(src/config.h)
 
 LOG4C_MAJOR_VERSION=1
-- 
2.0.0

